src/tasks/createGithubRelease/subtasks/getCreateReleaseTask.ts

```diff
@@ -38,72 +35,79 @@ export function getCreateReleaseTask({
       task.output = "Deleting existing release...";
       await github.deleteReleaseAndAssets(tag);
 
-      const contentHashPaths = await handleReleaseVariantFiles({
-        releaseDetailsMap,
-        composeFileName
-      });
-
       task.output = `Creating release for tag ${tag}...`;
-      await github.createRelease(tag, {
+      const releaseId = await github.createRelease(tag, {
         body: await getReleaseBody({ releaseDetailsMap }),
         prerelease: true, // Until it is actually published to mainnet
       });
 
-      // Clean content hash file so the directory uploaded to IPFS is the same
-      // as the local build_* dir. User can then `ipfs add -r` and get the same hash
-      contentHashPaths.map(contentHashPath => fs.unlinkSync(contentHashPath));
+      task.output = "Preparing release directories for Github release...";
+      prepareGithubReleaseFiles({ releaseDetailsMap, composeFileName });
+
+      task.output = "Uploading assets...";
+      await uploadAssets({ releaseDetailsMap, github, releaseId });
+
     }
   };
 }
 
-async function handleReleaseVariantFiles({
+function prepareGithubReleaseFiles({
```

Comment:
Creates `content-hash` file, compacts maifest and deletes build properties from compose